<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="3" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-09-19T09:50:08 IST" name="Suite" finished-at="2021-09-19T09:51:01 IST" duration-ms="53595">
    <groups>
    </groups>
    <test started-at="2021-09-19T09:50:08 IST" name="App Testing" finished-at="2021-09-19T09:50:08 IST" duration-ms="22">
    </test> <!-- App Testing -->
    <test started-at="2021-09-19T09:50:08 IST" name="Validating title in the Page" finished-at="2021-09-19T09:50:52 IST" duration-ms="43770">
      <class name="Academy.validateTitle">
        <test-method is-config="true" signature="initialize()[pri:0, instance:Academy.validateTitle@6865c751]" started-at="2021-09-19T09:50:08 IST" name="initialize" finished-at="2021-09-19T09:50:50 IST" duration-ms="42425" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method signature="validateAppTitle()[pri:0, instance:Academy.validateTitle@6865c751]" started-at="2021-09-19T09:50:50 IST" name="validateAppTitle" finished-at="2021-09-19T09:50:50 IST" duration-ms="53" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [FEATURED CO123URSES] but found [FEATURED COURSES]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [FEATURED CO123URSES] but found [FEATURED COURSES]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at Academy.validateTitle.validateAppTitle(validateTitle.java:44)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateAppTitle -->
        <test-method signature="validateHeader()[pri:0, instance:Academy.validateTitle@6865c751]" started-at="2021-09-19T09:50:51 IST" name="validateHeader" finished-at="2021-09-19T09:50:51 IST" duration-ms="43" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [An Academy to learn Everything about Testing] but found [AN ACADEMY TO LEARN EVERYTHING ABOUT TESTING]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [An Academy to learn Everything about Testing] but found [AN ACADEMY TO LEARN EVERYTHING ABOUT TESTING]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at Academy.validateTitle.validateHeader(validateTitle.java:57)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateHeader -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:Academy.validateTitle@6865c751]" started-at="2021-09-19T09:50:52 IST" name="teardown" finished-at="2021-09-19T09:50:52 IST" duration-ms="47" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- Academy.validateTitle -->
    </test> <!-- Validating title in the Page -->
    <test started-at="2021-09-19T09:50:08 IST" name="Validating navigation Bar" finished-at="2021-09-19T09:50:52 IST" duration-ms="43915">
      <class name="Academy.validateNavigationBar">
        <test-method is-config="true" signature="initialize()[pri:0, instance:Academy.validateNavigationBar@77b325b3]" started-at="2021-09-19T09:50:08 IST" name="initialize" finished-at="2021-09-19T09:50:52 IST" duration-ms="43708" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method signature="validateAppNavBar()[pri:0, instance:Academy.validateNavigationBar@77b325b3]" started-at="2021-09-19T09:50:52 IST" name="validateAppNavBar" finished-at="2021-09-19T09:50:52 IST" duration-ms="167" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateAppNavBar -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:Academy.validateNavigationBar@77b325b3]" started-at="2021-09-19T09:50:52 IST" name="teardown" finished-at="2021-09-19T09:50:52 IST" duration-ms="23" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- Academy.validateNavigationBar -->
    </test> <!-- Validating navigation Bar -->
    <test started-at="2021-09-19T09:50:08 IST" name="Validating User login" finished-at="2021-09-19T09:51:01 IST" duration-ms="53595">
      <class name="Academy.HomePage">
        <test-method is-config="true" signature="initialize()[pri:0, instance:Academy.HomePage@72bd06ca]" started-at="2021-09-19T09:50:08 IST" name="initialize" finished-at="2021-09-19T09:50:12 IST" duration-ms="4446" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method signature="basePageNavigation(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:Academy.HomePage@72bd06ca]" started-at="2021-09-19T09:50:12 IST" name="basePageNavigation" data-provider="getData" finished-at="2021-09-19T09:50:56 IST" duration-ms="43814" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[nonrestricteduser@qw.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Restrcited User]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- basePageNavigation -->
        <test-method signature="basePageNavigation(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:Academy.HomePage@72bd06ca]" started-at="2021-09-19T09:50:56 IST" name="basePageNavigation" data-provider="getData" finished-at="2021-09-19T09:51:01 IST" duration-ms="5173" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[restricteduser@qw.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[456788]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Non restricted user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- basePageNavigation -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:Academy.HomePage@72bd06ca]" started-at="2021-09-19T09:51:01 IST" name="teardown" finished-at="2021-09-19T09:51:01 IST" duration-ms="76" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- Academy.HomePage -->
    </test> <!-- Validating User login -->
  </suite> <!-- Suite -->
</testng-results>
